govcd/system_test.go

```diff
@@ -615,3 +615,19 @@ func (vcd *TestVCD) Test_GetStorageProfileByHref(check *C) {
 	check.Assert(foundStorageProfile, Not(Equals), types.VdcStorageProfile{})
 	check.Assert(foundStorageProfile, NotNil)
 }
+
+func (vcd *TestVCD) Test_GetOrgList(check *C) {
+
+	orgs, err := vcd.client.GetOrgList()
+	check.Assert(err, IsNil)
```

Comment:
check that `orgs` is not nill is missing

Reply:
done